bbb-mock.yml

```
@@ -0,0 +1,4 @@
+version: "2"
+services:
+  bbbmock:
```

```suggestion
  webserver:
    environment:
      MOODLE_DOCKER_BBB_MOCK: "true"
  bbbmock:
```

Let's declare the env variable here, so `config.php` is able to get it.